docs/reference/modules/node.asciidoc

```diff
@@ -68,8 +68,8 @@ A node that has the `master` role, which makes it eligible to be
 
 <<data-node,Data node>>::
 
-A node that has the `data` role. Data nodes hold data and perform data
-related operations such as CRUD, search, and aggregations. A node with the `data` role can fill any of the specialised data node roles.
+A node that has one of several data roles. Data nodes hold data and perform data
+related operations such as CRUD, search, and aggregations. A node with a generic `data` role can fill any of the specialised data node roles.
```

Comment:
```suggestion
related operations such as CRUD, search, and aggregations. A node with a generic `data` role can fill any of the specialized data node roles.
```

Reply:
+1 to preferring the American English spelling